sources/packages/api/src/services/application/application.service.ts

```diff
@@ -527,11 +535,67 @@ export class ApplicationService extends RecordDataModelService<Application> {
 
   /**
    * get all student applications.
+   * @param page, page number if nothing is passed then
+   * DEFAULT_PAGE_NUMBER is taken
+   * @param pageLimit, limit of the page if nothing is
+   * passed then DEFAULT_PAGE_LIMIT is taken
+   * @param sortField, field to be sorted
+   * @param sortOrder, order to be sorted
    * @param studentId student id .
    * @returns student Application list.
    */
-  async getAllStudentApplications(studentId: number): Promise<Application[]> {
-    return this.repo
+  async getStudentAppicationAndProcessDTO(
+    sortField: string,
+    studentId: number,
+    page = DEFAULT_PAGE_NUMBER,
+    pageLimit = DEFAULT_PAGE_LIMIT,
+    sortOrder = FieldSortOrder.ASC,
+  ): Promise<StudentApplicationAndCount> {
+    const applicationsAndCount = await this.getAllStudentApplications(
+      sortField,
+      studentId,
+      page,
+      pageLimit,
+      sortOrder,
+    );
+
+    return {
+      applications: applicationsAndCount[0].map((application: Application) => {
+        return {
+          applicationNumber: application.applicationNumber,
+          id: application.id,
+          studyStartPeriod: application.offering?.studyStartDate ?? "",
```

Comment:
Just FYI. this ?? operator is not supported in IE. For now IE is not one of the targeted browsers so we are good.
![image](https://user-images.githubusercontent.com/54600590/146419483-b0e6e2ab-8c0f-4d43-8c9b-64b2f5c81685.png)